src/data/sql-quiz.ts

```diff
@@ -1,4 +1,13 @@
 const sqlQuiz = [
+  {
+    Question: "Which clause set the child data to NULL when the parent data is deleted?",
+    Answer: "ON DELETE SET NULL",
+    Distractor1: "UPDATE TO NULL ",
+    Distractor2: "ON DELETE CASCADE",
+    Distractor3: "ALL OF THE ABOVE",
```

Comment:
we want to avoid language like all of the of the above because the answer are randomly sorted each time in the quiz

```suggestion
    Distractor3: "DELETE NULL",
```